libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -1825,7 +1949,7 @@
 			shellPath = /bin/sh;
 			shellScript = "\"$SRCROOT/../../shared/common/build/tools/generate_headers.sh\" -o \"$TARGET_BUILD_DIR/$PRODUCT_NAME.framework/Headers/$PRODUCT_NAME.h\" -u \"$SRCROOT/$TARGET_NAME/$TARGET_NAME.h\"";
 		};
-		CE4CE4FB1C0E6E07009F6029 /* ShellScript */ = {
+		E1C80D751C5C5056001B3A21 /* ShellScript */ = {
```

Comment:
@khawkins Adding the script for update_podspec_headers back as I was able to build successfully.